deployments/helm/nvidia-device-plugin/templates/daemonset-gfd.yml

```diff
@@ -182,6 +186,10 @@ spec:
             mountPath: "/etc/kubernetes/node-feature-discovery/features.d"
           - name: host-sys
             mountPath: "/sys"
+        {{- if typeIs "string" .Values.imexNodesConfigFile }}
```

Comment:
Should this also check for a non-empty value?

Reply:
suggestion taken